tests/suites/test_suite_pk.function

```diff
@@ -554,8 +554,9 @@ void pk_sign_verify_restart( int pk_type, int grp_id, char *d_str,
     cnt_restart = 0;
     do {
         ret = mbedtls_pk_sign_restartable( &prv, md_alg, hash, hlen,
-                                            sig, &slen, mbedtls_test_rnd_std_rand,
-                                            NULL, &rs_ctx );
+                                           sig, sizeof( sig ), &slen,
```

Comment:
Consistency nit: `sizeof sig`
Or `sizeof( sig )` in the other places

Reply:
Our [coding style](https://tls.mbed.org/kb/development/mbedtls-coding-standards) mandates parentheses around the argument of `sizeof`, but some existing code lacks parentheses. I've tried to keep each function consistent, but I'm not going to sweat over a few inconsistencies, especially since we're planning to change the coding style soon.